tests/libres_tests/res/enkf/test_programmatic_res_config.py

```diff
@@ -94,13 +96,11 @@ def setUp(self):
                 "ECLBASE": "SIM_KW",
                 "NUM_REALIZATIONS": 10,
                 "INSTALL_JOB": [
-                    {"NAME": "NEW_JOB_A", "PATH": "simulation_model/jobs/NEW_TYPE_A"},
-                    {"NAME": "NEW_JOB_B", "PATH": "simulation_model/jobs/NEW_TYPE_B"},
-                    {"NAME": "NEW_JOB_C", "PATH": "simulation_model/jobs/NEW_TYPE_C"},
+                    {"NAME": "NEW_JOB_A", "PATH": "NEW_TYPE_A"},
                 ],
                 "SIMULATION_JOB": [
                     {"NAME": "NEW_JOB_A", "ARGLIST": ["Hello", True, 3.14, 4]},
-                    {"NAME": "NEW_JOB_B", "ARGLIST": ["word"]},
+                    {"NAME": "NEW_JOB_A", "ARGLIST": ["word"]},
```

Comment:
Why the rename?

Reply:
Because I did not really see the added benefit of having two different simulation jobs when they were not testing different functionality. That saved writing one extra job file.